test/integration/src/test/java/io/pravega/test/integration/RestoreBackUpDataRecoveryTest.java

```diff
@@ -319,6 +345,37 @@ public void close() {
         }
     }
 
+
+    /**
+     * Creates a Pravega instance.
+     */
+    private static class PravegaRunner {
+        private final int containerCount;
+        private BookKeeperRunner bookKeeperRunner;
+        private SegmentStoreRunner segmentStoreRunner;
+        private ControllerRunner controllerRunner;
+
+        PravegaRunner(int instanceId, int bookieCount, int containerCount, StorageFactory storageFactory) throws Exception {
+            this.containerCount = containerCount;
+            this.bookKeeperRunner = new BookKeeperRunner(instanceId, bookieCount);
+            this.segmentStoreRunner = new SegmentStoreRunner(storageFactory, null, containerCount);
```

Comment:
These last 2 lines are the same as in `restart...` below. Replace them with an invocation of that method.

Reply:
Done.